src/anubhav/arrayexamples/StringsWithAtleastOneNumber.java

```diff
@@ -0,0 +1,34 @@
+package anubhav.arrayexamples;
+
+/*Program 2: Write a method which will return all String from given array which contains atleast one number. Main method should print the output.
+Input : {"TechnoCredits", "HP1w", "Pd3R4K9", "Krishna", "Aashvi25"}
+Output : HP1w Pd3R4K9 Aashvi25
+ */
+
+public class StringsWithAtleastOneNumber {
+
+	String[] findDigit(String[] temp) {
+		String[] output= new String[10];
```

Comment:
10 not be hard coded, you should use temp.length, as we can't know how many number will be sent by user